CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 7.70
 -----
-
+- A new button is added under Account Settings: "Change Avatar". It opens the in-app Safari with Gravatar's Avatar Quick Editor. [#1770]
```

Comment:
We changed our formatting a bit to make this into a link. Could you change this to:
```suggestion
- A new button is added under Account Settings: "Change Avatar". It opens the in-app Safari with Gravatar's Avatar Quick Editor. [#1770](https://github.com/Automattic/pocket-casts-ios/pull/1770)
```